velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -2943,6 +2943,56 @@ TEST_F(DateTimeFunctionsTest, dateParse) {
       dateParse("116", "%y+"), "Invalid format: \"116\" is malformed at \"6\"");
 }
 
+// The test for the alias now() is integrated into this test.
+TEST_F(DateTimeFunctionsTest, currentTimestamp) {
+  const auto evaluateCurrentTimestamp = [&](const std::string& functionName,
+                                            int64_t expectedTimestamp,
+                                            int16_t expectedTimeZone) {
+    // Pass an empty vector here since current_timestamp has no parameter.
+    auto emptyRowVector = makeRowVector(ROW({}), 1);
+
+    // Manually build the Velox expression to skip the DuckDB parsing.
+    auto callExpr = std::make_shared<const core::CallTypedExpr>(
+        TIMESTAMP_WITH_TIME_ZONE(),
+        std::vector<core::TypedExprPtr>{},
+        functionName);
+    auto actual = evaluate(callExpr, emptyRowVector);
+
+    auto resultTstz = actual->as<ConstantVector<ComplexType>>()
```

Comment:
Nit : Rename the next 3 variables as actualConstant, actualTimestamp and actualTimezone.